lib/puppeteer-helpers.js

```diff
@@ -122,6 +122,16 @@ function getAttribute ($element, attributeName) {
   return $element.evaluate((el, name) => el.getAttribute(name), attributeName)
 }
 
+/**
+ * Gets the accessible name of the given element, if it exists in the accessibility tree
+ *
+ * @param {import('puppeteer').ElementHandle} $element - Puppeteer element handle
+ * @returns {string} - The element's accessible name
```

Comment:
Looks like `page.accessibility.snapshot()` can return undefined

```suggestion
 * @returns {string | undefined} - The element's accessible name
```